ci/test/testinstall/install.go

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package testinstall
+
+import (
+	"github.com/magefile/mage/sh"
+)
+
+// TestInstallScript tests install.sh
+func TestInstallScript() error {
+	return sh.RunV("go", "test", "-v", "-tags=integration", "github.com/mysteriumnetwork/node/ci/test/testinstall")
```

Comment:
would be nice to have all e2e tests in `/e2e`

Reply:
I'll keep it separate because as they are now, e2e tests check node from functional perspective, and this merely tests the installation script.